contracts/contracts/bridge/IBridge.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache 2
```

Comment:
This can be ignored, does nothing at this point.

Reply:
This file or this line ?
If it's the file, should it just be deleted ?